democracy/views/section_comment.py

```diff
@@ -56,3 +58,74 @@ class SectionCommentViewSet(BaseCommentViewSet):
     model = SectionComment
     serializer_class = SectionCommentSerializer
     create_serializer_class = SectionCommentCreateSerializer
+
+
+class RootSectionCommentSerializer(SectionCommentSerializer):
+    hearing = serializers.SerializerMethodField()
+
+    class Meta(SectionCommentSerializer.Meta):
+        fields = SectionCommentSerializer.Meta.fields + ['hearing']
+
+    def get_hearing(self, section_comment):
+        return section_comment.section.hearing.id
+
+
+class CommentPagination(LimitOffsetPagination):
+    default_limit = 50
+
+
+class CommentFilter(filters.FilterSet):
+    hearing = django_filters.CharFilter(name='section__hearing__id')
+
+    class Meta:
+        model = SectionComment
+        fields = ['authorization_code', 'section', 'hearing']
+
+
+# root level (Section)Comment endpoint
```

Comment:
Well the point of the comment was that this is a comment endpoint, for which `SectionComment` objects are used. Clearly the comment was a fail as the first reader got it wrong :D